common/utils.go

```diff
@@ -1065,6 +1065,31 @@ func MarshalObjToYaml(obj runtime.Object) ([]byte, error) {
 	return b.Bytes(), nil
 }
 
+// StringifyMap stringifies the map values
+func StringifyMap(inputMap map[string]interface{}) map[string]interface{} {
```

Comment:
StringifyMap is a very specify function that is not a general usecase, so won't it be better to just place it in the collector or collect type package itself? or do we see this specific situation occurring in other places?

Reply:
done.